scheduler-interface-spec.md

```diff
@@ -569,9 +569,9 @@ message AllocationRelease {
   string partitionName = 1;
   // The application the allocation belongs to
   string applicationID = 2;
-  // The UUID of the allocation to release, if not set all allocations are released for
+  // The allocationID of the allocation to release, if not set all allocations are released for
   // the applicationID
-  string UUID = 3;
+  string allocationID = 3;
```

Comment:
See the change we did for groupID and reason: we reserve the ID and name for backwards compatibility